tests/codecs/test_json.py

```diff
@@ -0,0 +1,39 @@
+import pytest
```

Comment:
Is there a way to extend the tests for when `orjson` is and is not available? I am not sure which path the current tests exercise.

Reply:
The paths will use orjson if its available and otherwise the default json impl. Currently the only way to test this that i can see is to run the test twice with different environments.